src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable, AreaCalculable {
+    private final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
+
+    public abstract void draw();

```

Comment:
```suggestion

```

Reply:
not fixed
You are duplicating what is in the interfaces, what is the use of this, what have you done?